http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/api/ProxyRoutes.scala

```diff
@@ -60,14 +64,16 @@ trait ProxyRoutes extends UserInfoDirectives with CorsSupport with CookieHelper
                 // Proxy logic handled by the ProxyService class
                 // Note ProxyService calls the LeoAuthProvider internally
                 complete {
-                  val proxyFuture = proxyService.proxyRequest(userInfo, googleProject, clusterName, request)
                   // we are discarding the request entity here. we have noticed that PUT requests caused by
                   // saving a notebook when a cluster is stopped correlate perfectly with CPU spikes.
                   // in that scenario, the requests appear to pile up, causing apache to hog CPU.
-                  proxyFuture.failed.foreach { _ =>
-                    request.entity.discardBytes().future
-                  }
-                  proxyFuture
+                  for {
```

Comment:
Do you need the for-comp? Could this just be:
```
complete {
  proxyService.proxyRequest(userInfo, googleProject, clusterName, request).onError {
    case e =>
      IO(logger.warn(s"proxy request failed for ${userInfo} ${googleProject} ${clusterName}", e)) <* IO
        .fromFuture(IO(request.entity.discardBytes().future))
```
?

Reply:
ah probably not...will update